### PR TITLE
Itn manual

### PR DESCRIPTION
**Pull request by Christian-MK**
Per @peterVG 's request, I have created the first draft of the ITN manual. Content has been primarily pulled from Medium, articles and discord discussions.

### CODE REVIEW

File: docs/node-operation/itn-manual.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+ITN rewards cannot be earned without a valid license.
+```
+
+### 500,000 FACT deposit
+
+Participants need to maintain this balance in their Mainnet wallet, which will
+later need to be secured in a smart contract on the Testnet with tFACT
+(Testnet FACT).
+
+### Hardware and Infrastructure
```
Comment (ross-spencer) [inline docs/node-operation/itn-manual.md:96]: This is a good approach (some info here and a separate technical requirements section). 

Given that though, I'd be tempted to keep this information less detailed, and say something like:

* min requirements (phase 1) are, single CPU, 2GB ram.
* recommended requirements (phase 2 onwards) are 32GB ram, 8vcpu, 600GB disk.

For more information see the technical manual.

File: docs/node-operation/itn-manual.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+expected to operate more taxing software (eg a Cardano node, Ogmios, etc.).
+
+Participants should consult our
+[Software Requirements](technical-requirements#software) section while also
+anticipating that these requirements will change as development, and ITN phases,
+progress.
+
+### Knowledge requirements
+
+Participants must possess the skills to operate a Linux server and install
```
Comment (ross-spencer) [inline docs/node-operation/itn-manual.md:124]:
```suggestion
Participants must be able to run a Linux server and install
```

File: docs/node-operation/itn-manual.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+various software packages, including those provided by Orcfax. Additionally,
```
Comment (ross-spencer) [inline docs/node-operation/itn-manual.md:125]:
```suggestion
various software packages, including those provided by Orcfax.
```

File: docs/node-operation/itn-manual.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+participants should be capable of updating both the operating system and any
```
Comment (ross-spencer) [inline docs/node-operation/itn-manual.md:126]:
```suggestion
Participants should be capable of updating both the operating system and any
```

File: docs/node-operation/itn-manual.md [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+installed software packages to maintain security and functionality.
+
+While it is not a requirement at the outset, having an understanding of the
+Cardano node and its underlying protocols is highly recommended to facilitate a
```
Comment (ross-spencer) [inline docs/node-operation/itn-manual.md:130]: i wonder if this is more about an understanding of operating a cardano node?

we will be doing our utmost to wrap all this functionality and making the validators as appliance like as possible. (this is for our own sakes not just the ITN participants).

File: docs/node-operation/itn-manual.md [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+As participants accumulate rewards, they will be able to track them on our
+Validator Dashboard, and will be able to claim their accrued rewards at any
+given time.
+
+This approach prevents scheduled monthly releases that might affect price action
+and cause more volatility.
+
+To claim rewards:
+
+1. Send small amount of ADA to a given address (covering the minimum UTXO and
```
Comment (ross-spencer) [inline docs/node-operation/itn-manual.md:198]:
```suggestion
1. Send a small amount of ADA to a given address (covering the minimum UTXO and
```

File: docs/node-operation/itn-manual.md [hunk 7]
```diff
@@ -0,0 +1,210 @@
...
+
+This process can be repeated as often as a validator wishes to claim their
+rewards.
+
+#### Vesting method
+
+We are investigating whether the addition of a vesting method to the claiming
+process will be possible before ITN goes live. If we find a way to add this
+feature, we will implement it. However, we cannot guarantee at this moment that
+we will be able to do so.
```
Comment (ross-spencer) [inline docs/node-operation/itn-manual.md:210]: if we're not sure if we can do this, do we need to mention it at all? (or another way, who is asking for this?)

File: docs/node-operation/itn-manual.md (hunk 1 above)
Comment (ross-spencer) [reply docs/node-operation/itn-manual.md:96]: NB. we could also add a footnote that says something like: specs are based on the minimal viable digital ocean servers operated by Orcfax?

File: docs/node-operation/itn-manual.md [hunk 8]
```diff
@@ -0,0 +1,210 @@
+---
+id: ITN Manual
+sidebar_position: 4
+slug: /itn-manual
+---
+
+# Incentivized Testnet manual
```
Comment (ross-spencer) [inline docs/node-operation/itn-manual.md:7]: I wonder if it makes sense to call this an overview, and then have a separate manual section?

The manual section would just be a pointer to:

* manual ITN phase 1 <link>
* manual ITN phase 2 <link>
* manual ITN phase 2 <link>
* and so on...

And each of the manual pages would just have a title and then a message along the lines of "more info to be provided as we approach the ITN phase"?

**Review by ross-spencer — COMMENTED** [review]
looks like a great start Christian. Added some notes to cherry-pick from as desired.

File: docs/node-operation/itn-manual.md (hunk 8 above)
Comment (Christian-MK) [reply docs/node-operation/itn-manual.md:7]: @ross-spencer 
To be clear, are you suggesting a structure like this:
```
├─Introduction
├─Orcfax solution
├─Fact token
├─FAQ
├─ITN
    │   ├─ Overview // the current page being reviewed
    │   ├─ Phase 1 manual // link out to from overview "phase #" sections?
    │   ├─ Phase 2 manual
    │   ├─ Phase 3 manual
    │   ├─ Phase 4 manual
├─Node operation
└─Developer manual
```

File: docs/node-operation/itn-manual.md (hunk 7 above)
Comment (Christian-MK) [reply docs/node-operation/itn-manual.md:210]: Good point! I had pulled this out of a medium article but was unfamiliar with the context-- I think removing altogether is best.

File: docs/node-operation/itn-manual.md (hunk 8 above)
Comment (ross-spencer) [reply docs/node-operation/itn-manual.md:7]: That layout looks good!

File: docs/ITN/overview.md [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+The actual rewards accrued per validator will be proportional to their
+participation in a given day relative to all other validators.
+
+```text
+Example
+if a validator has collected 5% of all fact statements on a specific day,
+they will receive 5% of that day’s available rewards (after applying the ITN
+multiplier). This method ensures that optimal uptime and efficient validator
+setup will yield the highest possible rewards.
+```
```
Comment (ross-spencer) [inline docs/ITN/overview.md:112]: More idiomatic perhaps, might be to use the markup for quote:

> example
>
> if a validator has collected ...

File: docs/ITN/overview.md [hunk 10]
```diff
@@ -0,0 +1,153 @@
...
+Mainnet wallet. However, as we progress through the more advanced phases of the
+ITN, participants will also need one on Testnet.
+
+Holding a validator license on Mainnet will allow participants to mint one on
+Testnet.
+
+```text
+NB
+ITN rewards cannot be earned without a valid license.
+```
```
Comment (ross-spencer) [inline docs/ITN/overview.md:53]: Oh, this can be a quote as well...

> NB. ITN rewards cannot...

File: docs/node-operation/itn-manual.md (hunk 8 above)
Comment (ross-spencer) [reply docs/node-operation/itn-manual.md:7]: It looks like this page has been replaced with the overview now (and repeats much of the same information?), do we want to delete it?

**Comment by ross-spencer** [comment]
@Christian-MK took another look, it's looking great with the new sections. Is the  node operation folder to be deleted?

Only other comments I had were in places where you could use `>` quote  markdown to for nota-bene's and other extra text like examples. I only showed two places, but you might choose to do this and then look for other opportunities. 

Otherwise, looks good to merge and perhaps get some additional feedback from Vahid on through that lens, (i.e. as published docs)

File: docs/node-operation/itn-manual.md (hunk 8 above)
Comment (Christian-MK) [reply docs/node-operation/itn-manual.md:7]: a mindless hang-over
deleted!

**Comment by Christian-MK** [comment]
> Is the node operation folder to be deleted

should it be? I guess in my mind the two sections were distinct.

**Pull request merged**